moveit2_tutorials.repos

```diff
@@ -11,10 +11,10 @@ repositories:
     type: git
     url: https://github.com/ros-planning/moveit_msgs
     version: 2.2.0
-  moveit_msgs:
+  moveit_resources:
     type: git
-    url: https://github.com/ros-planning/moveit_msgs
-    version: 2.2.0
+    url: https://github.com/ros-planning/moveit_resources
+    version: ros2
```

Comment:
I think we need to use galactic branch here as well. Ideally we should release and use a tag again imo.
```suggestion
    version: galactic
```